src/zope/mkzeoinstance/tests/test_unix.py

```diff
@@ -229,14 +234,14 @@ def test_zeo_conf_content(self):
             "",
             "  # This logfile should match the one in the zeo.conf file.",
             "  # It is used by zdctl's logtail command, "
-                        "zdrun/zdctl doesn't write it.",
+            "zdrun/zdctl doesn't write it.",
```

Comment:
This is a list of strings, but one string in the middle of the list is too long and is split across two lines.
By removing the extra indentation you make it look like two list elements rather than one.

I suggest

```suggestion
            ("  # It is used by zdctl's logtail command, "
             "zdrun/zdctl doesn't write it."),
```

to make it apparent that this is one list item, split across two lines, without evoking flake8 warnings from unexpected indentation.

Reply:
That seems like a good idea